src/gateways.ts

```diff
@@ -84,4 +84,5 @@ export default [
   'https://ipfs.soul-network.com/ipfs/:hash',
   'https://ipfs.twdragon.net/ipfs/:hash',
   'https://getipfs.com/ipfs/:hash',
+  'https://storage.web3ph.dev/gw/:hash'
```

Comment:
trailing comma is required as it will prevent rebase hell
```suggestion
  'https://storage.web3ph.dev/gw/:hash',
```